src/Universe.cpp

```diff
@@ -1080,12 +1080,14 @@ void Lattice::setUniverses(int num_y, int num_x, Universe** universes) {
  */
 void Lattice::removeUniverse(Universe* universe) {
 
+  Universe* null = NULL;
+
   /* Set all locations in the array of universes array to NULL */
   for (int j=0; j < _num_y; j++) {
     for (int i = 0; i < _num_x; i++) {
       if (universe->getId() == getUniverse(i,j)->getId())
         _universes.at(j).push_back(std::pair<int, Universe*>
```

Comment:
While the change looks fine and won't change the existing routine, I don't understand what this routine does.  If some `Universe` in a `Lattice` matches the given one, we add a `NULL` pointer to a vector of X `Universes` at the corresponding Y coordinate?  How does that remove a `Universe`?